web/server/codechecker_server/api/report_server.py

```diff
@@ -563,20 +610,13 @@ def create_count_expression(report_filter):
         return func.count(literal_column('*'))
 
 
-def filter_report_filter(q, filter_expression, run_ids=None, cmp_data=None,
-                         diff_hashes=None):
-    if run_ids:
-        q = q.filter(Report.run_id.in_(run_ids))
-
+def filter_report_filter(q, filter_expression):
```

Comment:
Maybe we should rename this function to something more descriptive and add a docstring about why it is needed.
It just joins the File and ReviewStatus tables and applies the filter expression.